commit e92b250968e9656084ab5984689747ca615ff6e7
Author: Volker Wichmann <wichmann@laserdata.at>
Date:   Sun Mar 5 13:49:53 2017 +0100

    saga_api, CSG_Table::Del_Records(): bug fix, check record count correctly

diff --git a/src/saga_core/saga_api/table.cpp b/src/saga_core/saga_api/table.cpp
index 76a1d8d..fa1a66f 100644
--- a/src/saga_core/saga_api/table.cpp
+++ b/src/saga_core/saga_api/table.cpp
@@ -901,7 +901,7 @@ bool CSG_Table::Del_Record(int iRecord)
 //---------------------------------------------------------
 bool CSG_Table::Del_Records(void)
 {
-	if( m_Records > 0 )
+	if( m_nRecords > 0 )
 	{
 		_Index_Destroy();

